### ISSUE TITLE
Broken doc link in README.md

### ISSUE THREAD

**Issue by sara-02** [systers/sysbot#116]
## Description
As a contributor,
I need to update the broken link in the README.md,
so that I can view the correct deployment/or the correct service where the bot is hosted.

## Mocks
The line `This project is currently hosted here and will be shifted to paid servers soon.`, under the [main](https://github.com/systers/sysbot#systers-slackbot---sysbot-project) tag needs to be fixed.
Erroneous link [here]() when clicked lead to a 404.
![screenshot from 2018-07-25 20-18-04](https://user-images.githubusercontent.com/7047079/43208369-e7516f68-9047-11e8-959f-74f09eb9137b.png)

## Acceptance Criteria
Update the read with the correct link.

### Update [Required]
- [ ] README.md 

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
0.5 hours

**Comment by ramitsawhney27** [comment]
@sys-bot approve

**Comment by sammy1997** [comment]
Yes @aastha980 ! And the AWS URL is http://sysbot-dev.5uhcagxzd9.eu-central-1.elasticbeanstalk.com/ .
You can put both, but the second one has a bit of an issue

**Comment by sammy1997** [comment]
Yes @aastha980 ! And the AWS URL is http://sysbot-dev.5uhcagxzd9.eu-central-1.elasticbeanstalk.com/ .
You can put both, but the second one has a bit of an issue

**Comment by navyasri1** [comment]
http://sombuddha2016.pythonanywhere.com/ is not working. What is the current link? I want update and raise PR.
